go/packages/slslambda/slslambda.go

```diff
@@ -47,7 +48,16 @@ func CaptureError(ctx context.Context, err error) {
 		debugLog("capture error:", ctxErr)
 		return
 	}
-	span.captureError(err)
+	span.captureError(err, nil)
+}
+
+func CaptureErrorWithOptions(ctx context.Context, err error, options EventOptions) {
```

Comment:
Can now we change `CaptureError(ctx context.Context, err error)` function implementation to
```
CaptureErrorWithOptions(ctx, nil)
```
to not avoid duplicated code? And the same with `CaptureWarning`?

Reply:
Good call.